SimpleJavaProject/src/main/java/ana_raetcaia/Main.java

```diff
@@ -0,0 +1,13 @@
+package ana_raetcaia;
```

Comment:
can you try please to remove this file from this PR?
please note if you use the same branch for First task it will be removed there also, use another branch

Reply:
Removed first task.